airbyte-integrations/connectors/source-shopify/source_shopify/source.py

```diff
@@ -559,14 +716,17 @@ def streams(self, config: Mapping[str, Any]) -> List[Stream]:
             Transactions(config),
             BalanceTransactions(config),
             Pages(config),
+            MetafieldPages(config),
             PriceRules(config),
             DiscountCodes(config),
             Locations(config),
+            MetafieldLocations(config),
             InventoryItems(config),
             InventoryLevels(config),
             FulfillmentOrders(config),
             Fulfillments(config),
             Shop(config),
+            MetafieldShops(config)
```

Comment:
These class instances should be sorted alphabetically, as they were before.

Reply:
Done